extras/m_sqloper.postgresql.sql

```diff
@@ -1,9 +1,12 @@
 CREATE TABLE ircd_opers (
     id serial NOT NULL,
-    username text,
+    "name" text,
     "password" text,
-    hostname text,
+    "hash" text,
+    "host" text,
     "type" text,
+    "fingerprint" text,
+    autologin boolean NOT NULL DEFAULT 0,
```

Comment:
If you're going to quote these can you quote all of them.

Also, you could consider making the mandatory fields in all of the tables `NOT NULL`.